src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -508,39 +480,48 @@ class CosmasGoogleCloudService(
                             responseObserver: StreamObserver<CosmasProto.DeleteFileResponse>) = logging(
             "deleteFile", request.info.projectId, request.fileId,
             other = mapOf("fileName" to request.fileName)) {
-        val cemeteryName = "${request.info.projectId}-cemetery"
-        val cemeteryBytes: Blob? = try {
-            this.storage.get(getBlobId(cemeteryName, request.info))
+        try {
```

Comment:
I think we can safely delete this method, because we don't use it anymore.

Reply:
done